### PR TITLE
[MM-44531] - Updates to buttons in final confirmation step after purchase

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Update purchase confirmation step buttons and text

#### Ticket Link
[MM-44531](https://mattermost.atlassian.net/browse/MM-44531)

#### Screenshots

|  before  |  after  |
|----|----|
| 
Screenshot 2022-06-03 at 15 44 49
 |
Screenshot 2022-06-03 at 18 26 10
  |

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/purchase_modal/process_payment_setup.tsx [hunk 1]
```diff
@@ -112,7 +112,8 @@ export default class ProcessPaymentSetup extends React.PureComponent<Props, Stat
         if (subscribeCloudSubscription) {
             const productUpdated = await subscribeCloudSubscription(this.props.selectedProduct?.id as string);
 
-            if (!productUpdated) {
+            // the action subscribeCloudSubscription returns a true boolean when successful and an error when it fails
+            if (typeof productUpdated !== 'boolean') {
```
Comment (AGMETEOR) [inline components/purchase_modal/process_payment_setup.tsx:116]: Hey @pvev added you specifically to help review this change

**Comment by annelieseklein** [comment]
looks great @AGMETEOR ! 
1. Can we please have a max width of 568 pix for the text?
2. Let's update the "View billing" to be our current tertiary button
Screen Shot 2022-06-06 at 8 55 54 AM

4. "Workspace" should be the name of the team

**Comment by AGMETEOR** [comment]
> looks great @AGMETEOR !
> 
> 1. Can we please have a max width of 568 pix for the text?
> 2. Let's update the "View billing" to be our current tertiary button
> 
> Screen Shot 2022-06-06 at 8 55 54 AM
> 
> 3. "Workspace" should be the name of the team

@annelieseklein made requested changes
Screenshot 2022-06-06 at 20 55 15

**Comment by stevemudie** [comment]
@AGMETEOR I'm able to upgrade but all I get is a blank screen once the purchase goes through.

Refreshing at that point just drops me back in the System Console. Any ideas?

**Comment by AGMETEOR** [comment]
> @AGMETEOR I'm able to upgrade but all I get is a blank screen once the purchase goes through.
> 
> Refreshing at that point just drops me back in the System Console. Any ideas?

Hey @stevemudie , please test again. I have tested it myself and it works fine. Might have been a network issue.

**Review by pvev — APPROVED** [review]
LGTM! thanks @AGMETEOR :shipit:

**Review by nickmisasi — APPROVED** [review]

**Review by annelieseklein — APPROVED** [review]
great job!

**Comment by stevemudie** [comment]
@AGMETEOR All good except for a small wording issue.

The text reads: "You're are now upgraded to Cloud Professional" so the word "are" isn't needed.

**Comment by AGMETEOR** [comment]
@stevemudie fixed the typo

**Review by stevemudie — APPROVED** [review]
Looks great @AGMETEOR! Thanks you for your patience! ツ

**Pull request merged**